independent-projects/tools/pom.xml

```diff
@@ -149,16 +149,6 @@
                 <artifactId>maven-model-helper</artifactId>
                 <version>${maven-model-helper.version}</version>
             </dependency>
-            <dependency>
-                <groupId>com.fasterxml.jackson.core</groupId>
-                <artifactId>jackson-databind</artifactId>
-                <version>${jackson.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>com.fasterxml.jackson.dataformat</groupId>
-                <artifactId>jackson-dataformat-yaml</artifactId>
-                <version>${jackson.version}</version>
-            </dependency>
```

Comment:
Given how CI reacted, I don't think this is right. You should import the BOM instead and get rid of the versions.

Reply:
Ah wait, I just saw the BOM was at the bottom, we usually define it at the top.

Reply:
Yes, I was going to use the BOM but also saw it was a the bottom (which is odd). I can move it to the top.

Reply:
@jorsol would you mind pushing this cleanup in a separate PR? Just the removal of these two artifacts and moving the BOMs at the top.
The situation with Jackson looks extremely weird on the Gradle side and I'm not sure we will be able to push this very update.

Reply:
Yes, I could, but I honestly don't think that the cleanup is the problem.

Reply:
I totally agree with you. The problem is on the Jackson side AFAICS. Thus why I would like to have the cleanup in another PR so we can merge it right away.
As for the update, I don't know. I had the same issue in another Gradle based project so I'm pretty sure there's something wrong with this new version.